src/utils/Auth.js

```diff
@@ -25,25 +26,41 @@ export default class Auth {
     this.auth0.authorize();
   }
 
-  handleAuthentication(callback) {
+  loginAndLike(itemId, definitionId) {
+    const params = (itemId && definitionId) ? `?itemId=${itemId}&definitionId=${definitionId}` : '';
```

Comment:
i like that you use a queryparam here on the callback to auth0. I wonder if we can make it simple and just add our queryparams to the auth0 callback every time without conditionally checking

Reply:
Good idea. Removed loginAndLike().